src/config/index.js

```diff
@@ -40,6 +40,16 @@ import defaultConfigs from './defaults';
  * @param {function} config.module The module that will be decorated into the SDK
  */
 
+/**
+ * An object of interceptors that get called on every request or response.
+ * More information at {@link https://github.com/axios/axios#interceptors axios Interceptors}
+ *
+ * @typedef {Object} AxiosInterceptor
+ * @param {function} interceptor.fulfilled A function that is run on every successful request or
+ *   response
+ * @param {function} interceptor.rejected A function that is run on every failed request or response
+ */
+
```

Comment:
Similar question about this being `interceptor.request` and `interceptor.response`